setup.py

```diff
@@ -1,13 +1,34 @@
 from setuptools import setup
 
+with open('README.rst') as fh:
+    long_description = fh.read()
+
 setup(
     name='pynisher',
     version="0.6.3",
     packages=['pynisher'],
     install_requires=['docutils>=0.3', 'setuptools', 'psutil'],
+    extras_require={
+        "test": [
+            "pytest",
+            "pytest-xdist",
```

Comment:
Do we actually use xdist and timeout and flaky and mypy and scipy and matplotlib? Also, sklearn should probably be scikit-learn?

Reply:
I removed the unnecessary packages.